engine/src/test/java/com/sandboni/core/engine/result/ResultGeneratorTest.java

```diff
@@ -24,16 +24,17 @@ private Arguments getArguments(String fromChangeId, boolean selectiveMode, boole
             .runSelectiveMode(selectiveMode)
             .stage(stage.name())
             .runAllExternalTests(runAllExternalTests)
-            .enablePreview(enablePreview).build();
+            .enablePreview(enablePreview)
+            .ignoreUnsupportedFiles(unsupportedFile).build();
     }
 
-    private Processor getProcessor(String fromChangeId, boolean runSelective, boolean runAllExternal, Stage stage) {
-        return getProcessor(fromChangeId, runSelective, runAllExternal, stage, false, null);
+    private Processor getProcessor(String fromChangeId, boolean runSelective, boolean unsupportedFile, boolean runAllExternal, Stage stage) {
+        return getProcessor(fromChangeId, runSelective, unsupportedFile, runAllExternal, stage, false, null);
     }
 
-    private Processor getProcessor(String fromChangeId, boolean runSelective, boolean runAllExternal, Stage stage, boolean enablePreview, Finder[] finders) {
+    private Processor getProcessor(String fromChangeId, boolean runSelective, boolean unsupportedFile, boolean runAllExternal, Stage stage, boolean enablePreview, Finder[] finders) {
         return new ProcessorBuilder().with(procBuilder -> {
-            procBuilder.arguments = getArguments(fromChangeId, runSelective, runAllExternal, stage, enablePreview);
+            procBuilder.arguments = getArguments(fromChangeId, runSelective, unsupportedFile, runAllExternal, stage, enablePreview);
```

Comment:
same here. I think the parameter name should still be "ignoreUnsupportedFiles"..